dev

```diff
@@ -87,15 +108,21 @@ trap onexit EXIT INT
 
 function onexit {
   deleteServiceServer
-  docker service rm engine
+  docker service rm $MESG_SERVICE_NAME
 }
 
 docker service create \
-  --name engine \
+  --name $MESG_SERVICE_NAME \
   --tty \
   --label com.docker.stack.namespace=engine --label com.docker.stack.image=mesg/engine:$VERSION \
-  --mount type=bind,source=/var/run/docker.sock,destination=/var/run/docker.sock --mount type=bind,source=$HOME/.mesg,destination=/root/.mesg \
-  --network engine --publish 50052:50052 \
+  --env MESG_NAME=$MESG_NAME \
+  --env MESG_PATH=$MESG_PATH \
```

Comment:
```suggestion
```
The `MESG_PATH` must not be injected in the docker as its change the path in the Docker.
The destination path on line 124 is still `/root/.mesg`.